pkg/crc/config/viper_config_test.go

```diff
@@ -0,0 +1,182 @@
+package config
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/spf13/pflag"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	CPUs       = "cpus"
+	NameServer = "nameservers"
+)
+
+func newTestConfig(configFile, envPrefix string) (*Config, error) {
+	config, err := New(configFile, "CRC")
```

Comment:
@guillaumerose shouldn't this will be `New(configFile, envPrefix)` instead `CRC` hardcoded since this is used for each test functions?

Reply:
Fixed :+1: